sym-treemap-template.html

```diff
@@ -2,4 +2,5 @@
     <!--The treemap graphic will live within the following div-->
     <div class="exele-treemap"></div>
     <div class="exele-total-time"></div>
+    <table class="exele-attr-table"></table>
```

Comment:
I see that this code also builds the `<table>` element in code above, as well. Is there a way to only have it built once?